internal/cmd/oohelperd/internal/nwcth/internal.go

```diff
@@ -0,0 +1,51 @@
+package nwcth
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/ooni/probe-cli/v3/internal/iox"
+	"github.com/ooni/probe-cli/v3/internal/version"
+)
+
+const maxAcceptableBody = 1 << 24
```

Comment:
I think we need to clarify that this is the maximum acceptable body both for JSON and for webpages we fetch and we also need to justify why we have chosen this value.

Take a look at https://github.com/ooni/probe/issues/1727 for useful data to link to.